### ISSUE TITLE
Reboot Waltz server

### ISSUE THREAD

**Issue by lolux** [waltzio/waltz#233]
Starting approx 3–5 min. ago, Waltz is stuck in signed in mode even after I sign out of the Clef app. Does the server need a reboot?

**Comment by landakram** [comment]
Are you still seeing this issue? —

**Comment by lolux** [comment]
Yes :(

**Comment by jessepollak** [comment]
I just rebooted the logout workers on Clef's end, which may have fixed the issue. @lolux?

Obviously, this needs a bigger long term fix — will get on it when I return on Monday.

**Comment by josephwegner** [comment]
@jessepollak Thanks for restarting that.

Is that an issue on Clef's end, or is it something that Waltz is causing? If it is on Clef's end, do you know what the issue is?  Can we do anything to help you debug it?

**Comment by lolux** [comment]
Will check it when I get back later this evening...

**Comment by lolux** [comment]
... @jessepollak it is working now.

**Comment by jessepollak** [comment]
OK, so an update for y'all:

@landakram and I have been wrangling with our tasks queues (the problem which was causing this) over the last few days and we think we've figured it out. The solution involved updating some of the libraries we used (who woulda that a 3.0.1 would be "ancient" compared to a 3.1.7) and tweaking configuration a bit.

In our preliminary monitoring (over the last day), we've seen none of the issues we were seeing before, so we are thinking that this is resolved.
